doc/ref_model/refs.bib

```diff
@@ -348,3 +348,572 @@ @misc{anuket-abbreviatons
+}
+
+@misc{NIS,
+	title = {{The Security of Networks and Information Systems}},
+	url = {https://www.gov.uk/government/collections/nis-directive-and-nis-regulations-2018/}
+}
+
+@misc{NCSC,
+	title = {{The UK’s National Cyber Security Centre}},
+	url = {https://www.ncsc.gov.uk/}
+}
+
+@misc{data-protection,
+	title = {{Data Protection Act 2018}},
+	url = {https://www.legislation.gov.uk/ukpga/2018/12/contents/enacted/}
+}
+
+@misc{accc,
+	title = {{Australian Competition & Consumer Commission (ACCC)}},
+	url = {https://www.accc.gov.au/by-industry/telecommunications-and-internet}
+}
+
+@misc{ism,
+	title = {{Information Security Manual (ISM)}},
+	url = {https://www.cyber.gov.au/resources-business-and-government/essential-cyber-security/ism}
+}
+
+@misc{privacy-act-1988,
+	title = {{Privacy Act 1988}},
+	url = {https://www.oaic.gov.au/privacy/privacy-legislation/the-privacy-act}
+}
+
+@misc{NIST-SP-800-53,
+	title = {{NIST SP 800-53 Security and Privacy Controls for Federal Information Systems and Organizations}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-53r4.pdf}
+}
+
+@misc{NIST-SP-800-53A,
+	title = {{NIST SP 800-53A Assessing Security and Privacy Controls in Federal Information Systems and Organizations: Building Effective Assessment Plans}},
+	url = {https://serdp-estcp.mil/}
+}
+
+@misc{NIST-SP-800-63B,
+	title = {{NIST SP 800-63B Digital Identity Guidelines}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-63b.pdf}
+}
+
+@misc{NIST-SP-800-115,
+	title = {{NIST SP 800-115 Technical Guide to Information Security Testing and Assessment}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-115.pdf}
+}
+
+@misc{NIST-SP-800-125,
+	title = {{NIST SP 800-125 Guide to Security for Full Virtualization Technologies}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-125.pdf}
+}
+
+@misc{NIST-SP-800-125a,
+	title = {{NIST SP 800-125a Security Recommendations for Server-based Hypervisor Platforms}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-125Ar1.pdf}
+}
+
+@misc{NIST-SP-800-125b,
+	title = {{NIST SP 800-125b Secure Virtual Network Configuration for Virtual Machine (VM) Protection}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-125B.pdf}
+}
+
+@misc{NIST-SP-800-137,
+	title = {{NIST SP 800-137 Information Security Continuous Monitoring for Federal Information Systems and Organizations}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-137.pdf}
+}
+
+@misc{NIST-SP-800-145,
+	title = {{NIST SP 800-145 The NIST Definition of Cloud Computing}},
+	url = {https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf}
+}
+
+@misc{NIST-SP-800-190,
+	title = {{NIST SP 800-190 Application Container Security Guide}},
+	url = {https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-190.pdf}
+}
+
+@misc{NIST-SP-800-145,
```

Comment:
This is a duplicate entry, please remove it.

Reply:
Removed